lib/datasource/gradle-version/index.ts

```diff
@@ -50,10 +52,7 @@ export async function getPkgReleases({
   const allReleases: Release[][] = await Promise.all(
     versionsUrls.map(async url => {
       try {
-        const response: GradleRelease = await got(url, {
-          hostType: id,
-          json: true,
-        });
+        const response: GradleRelease = await http.getJson(url);
```

Comment:
`GradleRelease` type needs to be updated

Reply:
how/why?

Reply:
```suggestion
        const response = await http.getJson<GradleRelease[]>(url);
```

Can't provide a suggestion for this
```ts
interface GradleRelease {
    snapshot?: boolean;
    nightly?: boolean;
    rcFor?: string;
    version: string;
    downloadUrl?: string;
    checksumUrl?: string;
    buildTime?: string;
}
```